docs/getting-started.md

```diff
@@ -106,6 +106,18 @@ The result file will contain information about the `mime-types` package itself,
 information about each dependency. The scope names come from the package managers, for NPM packages these are usually
 `dependencies` and `devDependencies`, for Maven package it would be `compile`, `runtime`, `test`, and so on.
 
+Note that the `analyzer-result.yml` is supposed to capture all known information about a project, which can then be "filtered" in later steps. For example, scopes which are not relevant for the distribution will still be listed, but can be configured to get excluded so that they e.g. do not get downloaded and scanned by the _scanner_ step. To specify which scopes should be excluded, add an `.ort.yml` configuration file to the input directory of the analyzer. For more details see [Configuration File](config-file-ort-yml.md).
+
+In our case, we will add `[mime-types-path]/.ort.yml` with following content:
```

Comment:
@sschuberth should I change this as well?
"For this guide,  `[mime-types-path]/.ort.yml` is added with following content"

Reply:
Good point, yes. However, I'd change "is added" to something else as it sounds as if some tool wound add it for you. How about: "For this guide, `[mime-types-path]/.ort.yml` can be created with following content:"